src/sql/account.sql

```diff
@@ -1,10 +1,16 @@
-
 CREATE TABLE auth
-  (password TEXT PRIMARY KEY);
+  (method TEXT PRIMARY KEY,
+   extra_data TEXT);
 
 CREATE TABLE sessions
   (token TEXT PRIMARY KEY);
 
+CREATE TABLE pending_openid_requests
+  (state TEXT PRIMARY KEY,
+   code_verifier TEXT,
+   return_url TEXT,
+   expiry_time INTEGER);
+
```

Comment:
Sorry, I just saw your comment about this in the frontend PR. Here’s the code we use in the frontend: https://github.com/actualbudget/actual/blob/master/packages/loot-core/src/server/migrate/migrations.ts (you’d need to handle creating the migrations table, maybe in a new db file? (or this could be the time to merge the secrets database into the accounts database)

Reply:
Will look into this